lib/internal/bootstrap/web/exposed-wildcard.js

```diff
@@ -148,3 +97,18 @@ exposeLazyInterfaces(
   [
     'CompressionStream', 'DecompressionStream',
   ]);
+
+function createGlobalConsole() {
```

Comment:
Since this isn't going anywhere, you can just leave this in the original place (behind `TextEncoder` etc.)

Reply:
Thanks for catching this. Fixed.